modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-lang/src/main/resources/content/Language.properties

```diff
@@ -74,6 +74,7 @@ velocity=Velocity
 workflow-definition-name=Workflow Definition Name
 workflow-definition-status=Workflow Definition Status
 workflow-definition-version=Workflow Definition Version
+workflow-in-use-remove-assignement-to-x=Workflow in use. Remove its assignment to this asset: {0}.
```

Comment:
Hey @MariMalta, This changes(in the Language.properties file) must be added in a specific commit and the output of `gw buildLang` must be added in a separated "buildLang" commit. It's important to separate manual changes from the generated ones.

Thx

Reply:
all right , I'm going to change it.  Thx!